recipes/squirrel/all/conanfile.py

```diff
@@ -70,18 +78,29 @@ def package(self):
         cmake.install()
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "squirrel")
+        # CMakeDeps generator uses the global target if a downstream recipe depends on squirrel globally,
+        # and squirrel::squirrel is stolen by libsquirrel component (if shared) which doesn't depend on all other components.
+        # So this unofficial target is created as a workaround.
+        self.cpp_info.set_property("cmake_target_name", "squirrel::squirel-all-do-not-use")
```

Comment:
@czoido Do you know whether it's still true in `CMakeDeps`? I know it was an issue in `cmake_find_package*`.

Reply:
Yes, that is still true for `CMakeDeps` if the targets are independent and one steals the name like this one that will happen.